src/Weather/Skysight/Skysight.cpp

```
@@ -577,5 +606,21 @@ Skysight::DisplayActiveMetric(const TCHAR *const id)
 	      bdt.day, bdt.hour, bdt.minute);
   tstring label = desc.c_str();
 
+  auto *map = UIGlobals::GetMap();
+  if (map == nullptr)
+    return false;
+
+  LogFormat("Setting %s", path.c_str());
+  std::unique_ptr<MapOverlayBitmap> bmp;
+  try {
+    bmp.reset(new MapOverlayBitmap(path));
+  } catch (const std::exception &e) {
+    return false;
```

Why ignore the exception details?

---

Will add logging message.

---

But who will ever read the log?

---

remove try-catch.